changelog-extractor.py

```diff
@@ -0,0 +1,144 @@
+"""Script parses changelog from git based on 2 latest tags, formates changelog to .md format, creates/updates release on gitlab, posts to slack
+"""
+import subprocess
+import os
+
+def getMdFormattedChangelog(newCommit, oldCommit) : 
+    #saving parameters this way to make sure that they are read properly on bash command
+    get_log_format = '%x1f'.join(['%b', '%h']) + '%x1e'
+    bashCommand = 'git --no-pager --git-dir=.git log --first-parent --format="%s"' % get_log_format   + " "+ newCommit+"..." + oldCommit
```

Comment:
Some variables are in camel case and others in snake case, please pick only one, preferably snake case, according to python conventions

Reply:
done